backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -313,6 +314,21 @@ private ApiUser getAccountRequestApiUser() {
         });
   }
 
+  /** Only used for audit logging. */
+  public ApiUser getAnonymousApiUser() {
+    Optional<ApiUser> found = _apiUserRepo.findByLoginEmail(ANONYMOUS_EMAIL);
+    return found.orElseGet(
+        () -> {
+          ApiUser magicUser =
```

Comment:
What is the magic user exactly? Is this just a placeholder for logging anonymous requests?

Reply:
Yep, exactly. The audit log has `api user` as a required field in the database, so we need a pseudo-user to record anonymous requests.

Reply:
Makes sense - thanks!